src/language-js/parse/babel.js

```diff
@@ -45,6 +46,8 @@ const parseOptions = {
     "importReflection",
     "explicitResourceManagement",
     ["importAttributes", { deprecatedAssertSyntax: true }],
```

Comment:
Do we still need `deprecatedAssertSyntax`?

Reply:
Yes